### PR TITLE
[Tech Week] RAC + Spectrum + Tailwind examples

### PR DESCRIPTION
**Pull request by reidbarber**
Contains docs and more examples for how to create custom Spectrum components using RAC and Tailwind.

Can be previewed here: https://codesandbox.io/p/sandbox/silly-zhukovsky-m4s422?file=%2Fsrc%2FApp.js%3A1%2C1

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

### CODE REVIEW

File: examples/rac-spectrum-tailwind/src/components/GenInputField.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { useState } from "react";
+import { Input, Group, TextField, Button } from "react-aria-components";
+
+export function GenInputField() {
+  let [value, setValue] = useState("");
+  let [isTextFieldFocused, setIsTextFieldFocused] = useState(false);
+  return (
+    <Group
```
Comment (snowystinger) [inline examples/rac-spectrum-tailwind/src/components/GenInputField.tsx:8]: this example seems inverted from the docs https://react-spectrum.adobe.com/react-aria/Group.html#example

I think you could stop tracking the focused state if you flipped it around

File: examples/rac-spectrum-tailwind/src/components/PlanSwitcher.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+export function PlanSwitcher() {
+  return (
+    <RadioGroup
+      defaultValue="annual"
+      className="flex flex-col m-auto space-y-10 text-center"
+    >
+      <Label className="text-xl font-semibold mb-200">Plan Switcher</Label>
+      <div className="relative m-auto flex justify-evenly w-[400px]">
+        <span className="absolute py-1 text-xs text-white bg-gray-500 rounded left-100 px-75 -top-75">
```
Comment (snowystinger) [inline examples/rac-spectrum-tailwind/src/components/PlanSwitcher.tsx:30]: only if it's really easy, but might be cool for this badge to be inside the Option and contribute to the label via describedby

File: examples/rac-spectrum-tailwind/src/components/StarRatingGroup.tsx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+interface StarRatingGroupProps extends Omit<RadioGroupProps, "children"> {
+  ratingCount?: number;
+  value?: string;
+  defaultValue?: string;
+  onChange?: (value: string) => void;
+  isEmphasized?: boolean;
+  label?: string;
+}
+
+export function StarRatingGroup({
```
Comment (snowystinger) [inline examples/rac-spectrum-tailwind/src/components/StarRatingGroup.tsx:19]: i wish i could use this one with arrow keys

File: examples/rac-spectrum-tailwind/src/components/StarRatingGroup.tsx [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+                />
+              ))}
+            </div>
+          </Group>
+        </>
+      )}
+    </RadioGroup>
+  );
+}
+// TODO: Unset hover state
```
Comment (snowystinger) [inline examples/rac-spectrum-tailwind/src/components/StarRatingGroup.tsx:65]: still to do?

**Review by snowystinger — COMMENTED** [review]
very minor comments

File: examples/rac-spectrum-tailwind/src/components/PlanSwitcher.tsx (hunk 2 above)
Comment (reidbarber) [reply examples/rac-spectrum-tailwind/src/components/PlanSwitcher.tsx]: I ended up putting it inside the option, after the Annual text, and it has a good announcement.

File: examples/rac-spectrum-tailwind/src/components/StarRatingGroup.tsx (hunk 3 above)
Comment (reidbarber) [reply examples/rac-spectrum-tailwind/src/components/StarRatingGroup.tsx:20]: Fixed, thanks!

File: examples/rac-spectrum-tailwind/src/components/GenInputField.tsx (hunk 1 above)
Comment (reidbarber) [reply examples/rac-spectrum-tailwind/src/components/GenInputField.tsx:8]: I wanted to have the focus ring around the entire bar if the input was focused, and just around the button if it is focused. I think that's different behavior than what you're talking about (I may be wrong).

**Review by snowystinger — APPROVED** [review]
Happy for followup, in general it looks good. Do you have a new link for the codesandbox? i still can't use arrow keys to navigate the star rating in the link in the description

**Comment by reidbarber** [comment]
> i still can't use arrow keys to navigate the star rating in the link in the description

@snowystinger Can you try refreshing? It seems to be working for me. Let me know if it still doesn't work and I can investigate.

**Review by yihuiliao — APPROVED** [review]
lgtm! also just gonna pop in and say that i'm able to use the arrow keys to navigate the star rating if that's helpful at all

**Pull request merged**